org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/debug/node/NodeRunDAPDebugDelegate.java

```diff
@@ -99,4 +110,124 @@ public void launch(ILaunchConfiguration configuration, String mode, ILaunch laun
 		}
 	}
 
+	private boolean configureAdditionalParameters(Map<String, Object> param) {
+		String program = (String)param.get(PROGRAM);
+		String cwd = (String)param.get(CWD);
+		
+		if (program != null && program.endsWith(".ts")) {
```

Comment:
It's IMO better to use content-type than directly using file name patterns, as content-types allow much more control, flexibility, inheritance...
`Platform.getContentTypeManager().getContentType("org.eclipse.wildwebdeveloper.ts").isAssociatedWith(new Path(program).getName())`

Reply:
Sure